src/transformers/training_args.py

```diff
@@ -696,6 +699,11 @@ class TrainingArguments:
             for instruction fine-tuning. Check out the [original paper](https://arxiv.org/abs/2310.05914) and the
             [original code](https://github.com/neelsjain/NEFTune). Support transformers `PreTrainedModel` and also
             `PeftModel` from peft.
+        galore_target_modules (`List[str]`, *optional*):
+            The GaLoRe target modules, i.e. the module names that you would like to train, using GaLoRe algorithm
+            https://arxiv.org/abs/2403.03507
+            See: https://github.com/jiaweizzhao/GaLore for more details. You need to make sure to pass a valid GaloRe
+            optimizer, e.g. one of: "galore_adamw", "galore_adamw_8bit", "galore_adafactor"
```

Comment:
Worth mentioning that only `Linear` layers work?

Reply:
yes!

Reply:
Done!